src/containers/BooksList.js

```diff
@@ -37,13 +37,13 @@ const renderList = ({ state, handleRemoveBook, handleFilterChange }) => {
       </div>
       <div className="table-wrapper">
         <table className="fl-table">
-          <thead>
+          {/* <thead>
             <tr>
               <th>BookId</th>
               <th>Category</th>
               <th>Title</th>
             </tr>
-          </thead>
+          </thead> */}
```

Comment:
- [ ] [REQUIRED] Inline comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history.